### DISCUSSION TITLE
Making slides available somewhere?

### DISCUSSION THREAD

**Discussion by kkmann** [openstatsware/website#25] [Ideas]
@danielinteractive hope the talk went well! Any chance we could make the (build) quarto slides available in the blog as well?

**Comment by danielinteractive** [comment]
@kkmann The talk will only happen on 8 Dec - at that point I will make them available. Blog entry is a good idea, thanks!
As mentioned in the PR, I uploaded them now already to be available for the other speakers in the session, as asked for by the organizers.
